trustpinning/certs.go

```diff
@@ -161,6 +161,7 @@ func ValidateRoot(prevRoot *data.SignedRoot, root *data.Signed, gun string, trus
 		logrus.Debugf("failed to verify TUF data for: %s, %v", gun, err)
 		return nil, &ErrValidationFail{Reason: "failed to validate integrity of roots"}
 	}
+	signedRoot.Signatures = root.Signatures
```

Comment:
Good thought. Given the current use I'd be OK with `IsValid` only working correctly for delegations if it's not straightforward to fix for other roles too. We can file an issue to track implementing it for other roles.